src/PrestaShopBundle/Form/Admin/Sell/Product/Description/DescriptionType.php

```diff
@@ -93,18 +105,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label' => $this->trans('Summary', 'Admin.Global'),
                 'type' => FormattedTextareaType::class,
                 'options' => [
-                    'limit' => ProductSettings::MAX_DESCRIPTION_SHORT_LENGTH,
+                    'limit' => $shortDescriptionLimit,
                     'attr' => [
                         'class' => 'serp-default-description',
                     ],
                     'constraints' => [
                         new Length([
-                            'max' => ProductSettings::MAX_DESCRIPTION_SHORT_LENGTH,
+                            'max' => $shortDescriptionLimit,
```

Comment:
You should use `TinyMceMaxLength` instead of `Length` here, this should fix the bug you mentioned in your comment (and for description field as well)

Reply:
Actually I'm not even sure the Length constraint is required since `FormattedTextareaType` seems to automatically add the `TinyMceMaxLength` as long as `limit` is correctly set (to be tested)

Reply:
aha, it didn't work. I fixed it with recent commits using the normalizer in options resolver to set default "constraints" value. Feedback appreciated

Reply:
also constraint was setin view->vars. It didn't seemed to be used, im not sure if I didn't miss anything, but I removed that part.